### PR TITLE
Update action workflows

### PR DESCRIPTION
**Pull request by tortmayr**
#### What it does

Update action workflows
#### Changelog

-   [ ] This PR should be mentioned in the changelog
-   [ ] This PR introduces a breaking change (if yes, provide more details below for the changelog and the migration guide)

### CODE REVIEW

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -12,11 +12,7 @@
 name: 'CodeQL'
 
 on:
-  push:
-    branches: [master]
-  pull_request:
-    # The branches below must be a subset of the branches above
-    branches: [master]
+  workflow_dispatch:
```
Comment (martin-fleck-at) [inline .github/workflows/codeql-analysis.yml:15]: Are we running the CodeQL not on PRs and master anymore?

**Review by martin-fleck-at — APPROVED** [review]
Change looks good to me in general, just not sure why we change the CodeQL workflow.

File: .github/workflows/codeql-analysis.yml (hunk 1 above)
Comment (tortmayr) [reply .github/workflows/codeql-analysis.yml]: Fixed

**Review by martin-fleck-at — APPROVED** [review]
LGTM!

**Pull request merged**